buildpipeline/windows.groovy

```diff
@@ -74,13 +74,11 @@ simpleNode('Windows_NT','latest') {
                 {
                     targetHelixQueues = ['Windows.10.Amd64.Open',
                                          'Windows.7.Amd64.Open',
-                                         'Windows.81.Amd64.Open',]
+                                         'Windows.81.Amd64.Open',
+                                         'Windows.10.Amd64.ClientRS3.ES.Open',]
```

Comment:
Do we want to get rid of the default Windows.10.Amd64.Open? It should be the same as the Windows.10 in spanish.

cc: @danmosemsft

Reply:
@danmosemsft  why its a good idea to get rid of English Language Build ?